File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/filtri/filtroCalcoloNPK.ts
import { FaseCicloColturale } from "../metaschema/fase";
import { FinalitaPianoConcimazione } from "../metaschema/FinalitaPianoConcimazione";
import { RegolamentoConcimazione } from "../metaschema/RegolamentoConcimazione";
import { GruppoFinalita } from "../metaschema/utilizzi/GruppoFinalita";
import { Specie } from "../metaschema/utilizzi/Specie";

export class FiltroCalcoloNPK{
    regolamento: RegolamentoConcimazione;
    specie: Specie;
    finalita: FinalitaPianoConcimazione;
    stato: FaseCicloColturale;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreDTOStd/InData/Metaschema/FiltroCalcoloNPK.cs
using AgronicaCoreModelsSTD.metaschema;
using AgronicaCoreModelsSTD.metaschema.utilizzi;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreDTOStd.InData.Metaschema
{
    public class FiltroCalcoloNPK
    {
        public RegolamentoConcimazione regolamento { get; set; }
        public Specie specie { get; set; }
        public FinalitaPianoConcimazione finalita { get; set; }
        public FaseCicloColturale stato { get; set; }
    }
}
